### PR TITLE
docs: textarea for reproduce, logs and sth else

### PR DESCRIPTION
**Pull request by SaarChaffee**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -16,6 +16,23 @@ body:
...
+      required: false
+        
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: Text
+    validations:
+      required: false
```
Comment (MaikoTan) [inline .github/ISSUE_TEMPLATE/bug_report.yml:35]: I think it is better to put this section on after `expected`, as this might be extremely long.

**Comment by MaikoTan** [comment]
There is no a type named `doc`, there's only `docs`.
For more information, see: https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional#type-enum

**Comment by MaikoTan** [comment]
Why is it closed

**Comment by SaarChaffee** [comment]
> Why is it closed

maybe my account has its own consciousness

**Pull request closed without merging**